adventure/adventure.py

```
@@ -3938,6 +4032,172 @@ async def bless(self, ctx: Context):
                         ),
                     )
 
+    @commands.command()
+    @commands.guild_only()
+    async def insight(self, ctx: Context):
+        """[Psychic Class Only]
+        This allows a Psychic to expose the current enemy's weakeness to the party.
+        """
+        try:
+            c = await Character.from_json(self.config, ctx.author, self._daily_bonus)
+        except Exception:
+            log.exception("Error with the new character sheet")
```

do we want to return the exception here too for visibility?

---

Not needed, using `log.exception` will auto print the traceback